services/order.py

```diff
@@ -0,0 +1,45 @@
+from datetime import datetime
+
+from django.contrib.auth import get_user_model
+
+from django.db import transaction
+
+from django.db.models import QuerySet
+
+from db.models import Order, User, Ticket, MovieSession
+
+
+def create_order(
+        tickets: list[Ticket],
+        username: str | User,
+        date: datetime = None
+) -> Order:
+    with transaction.atomic():
+        users = get_user_model().objects.get(username=username)
+        order_data = {"user": users}
+        Order.objects.create(**order_data)

```

Comment:
You should save created order into varible and utilize it instead of getting it on the line 29